build/index.js

```diff
@@ -276,7 +277,7 @@ async function main() {
 
     const eslintOptions = ["--ext", ".ts,.js,.tsx,.jsx", "."];
 
-    if (!["list", "list-deps", "clean"].includes(command)) {
+    if (!["list", "list-deps"].includes(command)) {
```

Comment:
This change, and the one below, were necessary for `clean` to work, now that it's using `shx`,  a root module. *I still think it's a good idea to use shx for the reasons I wrote down elsewhere*, but I guess I know someone who's not gonna like this 🙈